examtool/examtool/cli/deploy.py

```diff
@@ -67,9 +67,10 @@ def deploy(exam, json, roster, start_time, enable_clarifications):
     print("Exam deployed to https://exam.cs61a.org/{}".format(exam))
 
     print("Initializing announcements...")
+    idf = IDFactory()
     elements = list(extract_questions(exam_content, include_groups=True))
     for element in elements:
-        element["id"] = element.get("id", rand_id())  # add IDs to groups
+        element["id"] = element.get("id", idf.get_id())  # add IDs to groups
```

Comment:
now we can use `rand_id` again

Reply:
I think this is still fine to keep just because it will make sure the keys are unique.

Reply:
oh nvm Ill change it back.